pkg/console/install_panels.go

```diff
@@ -741,8 +741,9 @@ func addNetworkPanel(c *Console) error {
 		if hostName == "" {
 			return "must specify hostname", nil
 		}
-		if errs := validation.IsQualifiedName(hostName); len(errs) > 0 {
-			return fmt.Sprintf("%s is not a valid hostname", hostName), nil
+		// ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go#L242-L245
```

Comment:
nit: Use a permalink for the ref in case the upstream is changed

Reply:
Updated it. Thank you.